### PR TITLE
Updated notebooks to make them more "general"

### PR DESCRIPTION
**Pull request by BillmanH**
I created a group of notebooks that I think cover the "Analyst Experience". The 4 notebooks take you step by step through the process of creating twins/models/ and relationships.

### CODE REVIEW

**Comment by elhorton** [comment]
Hi @mayurid @lmazuel and @annatisch , @zolvarga and I have talked with Bill about adding these notebooks as samples for the ADT SDK as they showcase the analyst use case, could you please help us get these merged or let us know of any changes you suggest? Thanks!

File: sdk/digitaltwins/azure-digitaltwins-core/samples/notebooks/models/area.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+    "@id": "dtmi:billmanh:area;1",
```
Comment (annatisch) [inline sdk/digitaltwins/azure-digitaltwins-core/samples/notebooks/models/area.json:2]: Could we select more generic IDs for the various models?

File: sdk/digitaltwins/azure-digitaltwins-core/samples/notebooks/01 Patrons.ipynb [hunk 2]
```diff
@@ -0,0 +1,603 @@
...
+    "[Here is the doc on the query language](https://docs.microsoft.com/en-us/azure/digital-twins/concepts-query-language)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Connecting\n",
+    "\n",
+    "**note**: you will need to replace {`home-test-twin.api.wcus.digitaltwins.azure.net`} with the address of your ADT. "
```
Comment (annatisch) [inline sdk/digitaltwins/azure-digitaltwins-core/samples/notebooks/01 Patrons.ipynb:29]: Is there anyway we can configure this to pull the URL from an environment variable? Or some other configurable source?

**Comment by annatisch** [comment]
Thank you @BillmanH, @elhorton - this is awesome - thank you for contributing samples!
@czubair, @maggiepint - do we have any guidance on adding these kinds of more comprehensive samples into the repo? Do we typically keep them in this repo, or elsewhere?

File: sdk/digitaltwins/azure-digitaltwins-core/samples/notebooks/01 Patrons.ipynb [hunk 3]
```diff
@@ -0,0 +1,603 @@
+{
```
Comment (annatisch) [inline sdk/digitaltwins/azure-digitaltwins-core/samples/notebooks/01 Patrons.ipynb:1]: Could these files be renamed to remove space characters? e.g. `01_patrons.ipynb` or `01-Patrons.ipynb`

File: sdk/digitaltwins/azure-digitaltwins-core/samples/notebooks/01 Patrons.ipynb (hunk 2 above)
Comment (BillmanH) [reply sdk/digitaltwins/azure-digitaltwins-core/samples/notebooks/01 Patrons.ipynb:29]: I could, however that would require extra steps from the analyst. AAD protects access to the site so it isn't a security issue (at least not on my side). 

My goal is to create notebooks that an analyst can use to quickly get up to speed on the process. Outside of setting up a conda environment, I would want to minimize the end-user setup.

File: sdk/digitaltwins/azure-digitaltwins-core/samples/notebooks/01 Patrons.ipynb (hunk 3 above)
Comment (BillmanH) [reply sdk/digitaltwins/azure-digitaltwins-core/samples/notebooks/01 Patrons.ipynb:1]: Sure I'll do that.

File: sdk/digitaltwins/azure-digitaltwins-core/samples/notebooks/models/area.json (hunk 1 above)
Comment (BillmanH) [reply sdk/digitaltwins/azure-digitaltwins-core/samples/notebooks/models/area.json:2]: Sure I can do that.

Comment (BillmanH) [reply sdk/digitaltwins/azure-digitaltwins-core/samples/notebooks/models/area.json]: Changed it to `mymodels`

File: sdk/digitaltwins/azure-digitaltwins-core/samples/notebooks/01 Patrons.ipynb (hunk 3 above)
Comment (BillmanH) [reply sdk/digitaltwins/azure-digitaltwins-core/samples/notebooks/01 Patrons.ipynb]: removed spaces from the notebooks

File: sdk/digitaltwins/azure-digitaltwins-core/samples/notebooks/01 Patrons.ipynb (hunk 2 above)
Comment (BillmanH) [reply sdk/digitaltwins/azure-digitaltwins-core/samples/notebooks/01 Patrons.ipynb]: As a compromise, I changed this to:
```python
# you will get this from the ADT resource at portal.azure.com
your_digital_twin_url = "home-test-twin.api.wcus.digitaltwins.azure.net"

azure_cli = AzureCliCredential()
service_client = DigitalTwinsClient(
    your_digital_twin_url, azure_cli)
service_client
```

**Comment by BillmanH** [comment]
@elhorton, @czubair, @maggiepint,  On thinking about this Is this the best place for notebooks? I was looking at some other Azure repositories and saw that they have separate repos for notebooks. Is that something you can set up for digital twin? Here's an example from [azure machine learning](https://github.com/Azure/MachineLearningNotebooks). It may be better to attach them in that way. What do you think?

**Comment by elhorton** [comment]
> @elhorton, @czubair, @maggiepint, On thinking about this Is this the best place for notebooks? I was looking at some other Azure repositories and saw that they have separate repos for notebooks. Is that something you can set up for digital twin? Here's an example from [azure machine learning](https://github.com/Azure/MachineLearningNotebooks). It may be better to attach them in that way. What do you think?

Adding in my thoughts-- I think for now, since the Jupyter notebooks run the Python ADT SDK, this repo would be a good place for them as they're high quality samples and most discoverable here. However, if @maggiepint or @czubair don't think the Azure SDK repo is a good location, then I absolutely think we should set up a generic ADT samples repo-- I'd collaborate with the ADT team to help get that set up. @maggiepint @czubair @annatisch do you have thoughts on this? let us know if it's easier to chat this through.

**Comment by elhorton** [comment]
@maggiepint or @czubair , can you please help with this? Would love to get this merged. Thanks.

**Review by lmazuel — APPROVED** [review]

**Comment by lmazuel** [comment]
Even if we're unsure of the right place, I prefer merge the PR than to let it open. We can move the files as necessary. @annatisch CC

**Pull request merged**